docs/deployment/containers/docker.txt

```diff
@@ -223,12 +223,21 @@ Here's how you would mount ``crate.yml``::
 Here, ``/srv/crate/config/crate.yml`` is an example path, and should be
 replaced with the path to your host machine ``crate.yml`` file.
 
-Healthcheck
------------
+Troubleshooting
+===============
 
-The official `CrateDB Docker image`_ ships with a configured liveness
-`Docker healthcheck`_. This `Docker healthcheck`_ will flag that the CrateDB
-process might have crashed or hanged inside the container without terminating.
+The official `CrateDB Docker image`_ ships with a liveness `healthcheck`_
+configured. This healthcheck will flag a problem when the CrateDB process might
+have crashed or hanged inside the container without terminating.
+
+If you use `Docker Swarm`_ and you are experiencing trouble starting your Docker
+containers, try to deactivate the health check.
+
+You can do that like so:
+
+.. code-block:: yaml
+
+    disable: true
```

Comment:
Mm, this is unclear. I'd either include the full path down to the setting:
```
services:
 service:
    healthcheck:
     disable: true
```

Or just:
```
healthcheck:
  disable: true
```